src/pages/directory/[clubId].tsx

```diff
@@ -34,16 +34,14 @@ import {
   type InferGetStaticPropsType,
   type GetServerSidePropsContext,
 } from 'next';
-import DbProvider from '../../backend_tools/db_provider';
 import { createServerSideHelpers } from '@trpc/react-query/server';
 import { appRouter } from '@src/server/api/root';
 import { createInnerTRPCContext } from '@src/server/api/trpc';
 import { api } from '@src/utils/api';
+import { db } from '@src/server/db';
 
 export const getStaticPaths: GetStaticPaths = async () => {
-  const db = new DbProvider();
-  const clubs = await db.getAllClubs();
-
+  const clubs = await db.query.club.findMany();
```

Comment:
Is there a reason you're making a direct call to the db instead of through trpc (`const clubs = api.club.all()`)?

Reply:
The trpc `api` object is for use within react components. `getStaticPaths` happens at compile time so that's why we're using it that way

Reply:
👍